libsolidity/formal/CHC.cpp

```diff
@@ -601,7 +601,12 @@ bool CHC::shouldVisit(ContractDefinition const& _contract) const
 
 bool CHC::shouldVisit(FunctionDefinition const& _function) const
 {
-	return _function.isImplemented();
+	if (
+		_function.isPublic() &&
```

Comment:
Not that it's too relevant, but: ``return _function.isPublic() && _function.isImplemented();`` :-)?